exporters/otlp/trace/src/test/java/io/opentelemetry/exporter/otlp/trace/TraceMarshalerTest.java

```diff
@@ -125,6 +140,39 @@ void marshalAndSizeRequest_InstrumentationLibrary() throws IOException {
             testSpanDataWithInstrumentationLibrary(InstrumentationLibraryInfo.create("", ""))));
   }
 
+  @RepeatedTest(10000)
```

Comment:
do we want 10,000 for CI? Any rough guess how long this takes to run?

Reply:
Oops, was drift back to 100